verified-reviews/utils/client.ts

```diff
@@ -96,13 +102,13 @@ export const createClient = (params: ConfigVerifiedReviews | undefined) => {
   };
   /** @description https://documenter.getpostman.com/view/2336519/SVzw6MK5#daf51360-c79e-451a-b627-33bdd0ef66b8 */
   const reviews = (
-    { productId, count = 5, offset = 0, order: _order = "date_desc" }:
+    { productId, count = 5, offset = 0, order: _order = "date_desc", customizeOrder = false }:
       & PaginationOptions
       & {
         productId: string | string[];
       },
   ) => {
-    const order = orderMap[_order];
+    const order = customizeOrder ? orderMap[_order as keyof typeof orderMap] : _order;
```

Comment:
```suggestion
    const order = customizeOrder ? _order : orderMap[_order as keyof typeof orderMap];
```

as default, keep current behavior.